src/vuestic-theme/vuestic-components/va-button/VaButton.vue

```diff
@@ -325,7 +325,7 @@ export default {
         background-color: $focus-color;
         color: $border-color !important;
 
-        .va-button-toggle & {
+        .va-button-toggle &, .va-pagination & {
           background-color: $border-color;
```

Comment:
Yes, we need it, because standard button group (as a group of default buttons) doesn't support specific focus/active color, which we use in button toggle component.
See button group demo as a proof.